LaravelMedia/Rules/Concerns/GetsMimeTypeFromBase64.php

```diff
@@ -39,4 +39,15 @@ protected static function removeScheme(string $value): string
 
         return $value;
     }
+
+    /**
+     * Returns the size of a base64 string in kilobytes.
+     *
+     * @param  string  $value
+     * @return float
+     */
+    protected function getSize(string $value): float
```

Comment:
Looks a little weird that this method is located in a trait called `GetsMimeType...`

Reply:
Rename it to `ValidatesBase64`.